tests/pebble.rs

```diff
@@ -184,6 +184,17 @@ fn try_tracing_init() {
         .try_init();
 }
 
+fn dns_identifiers<I, T>(dns_names: I) -> Vec<Identifier>
```

Comment:
Nits: move this down below its user? Also use `impl Trait` to remove the indirection, I think we could write `names: impl IntoIterator<Item = impl ToString>`?

Reply:
Done x2.